daisy_workflows/build-publish/rhui/cds_artifacts/answers.yaml

```diff
@@ -1,11 +1,39 @@
+artifact_dir: /var/lib/rhui/remote_share/pulp3/artifact
+cds_certs_expiration: '7300'
```

Comment:
seconds?

Reply:
this is an irrelevant field for us - it would set the default days for a CDS cert generated by rhui-manager on this RHUA. we don't generate certs that way.